ci/coverage-report-generator/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Script to run generate_report.rb within docker so that the gems don't
+# need to be installed locally.
+# generate_report.rb generates an html report from simplecov json output.
+
+set -xeu
+
+repo_root=$(git rev-parse --show-toplevel)
+# Use the first arg, or if not supplied use the simplecov default report location
+report_file="${1:-${repo_root}/coverage/.resultset.json}"
+image="ruby:2.6.5-stretch"
```

Comment:
nit: Maybe pull this up to make sure it's visible since it's not the same ruby version that Conjur natively runs